pkg/clients/common.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 The Kubernetes Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
+import (
+	"context"
+
+	cosi "sigs.k8s.io/container-object-storage-interface-spec"
```

Comment:
Should this be attempting to use new monorepo?

```suggestion
	cosi "sigs.k8s.io/container-object-storage-interface/proto"
```

Reply:
I'll fix that in future, we need to fix the import paths in the core repo first.